ufc/tests/test-case-kill-switch-flag.json

```diff
@@ -13,6 +13,11 @@
       "subjectAttributes": { "email": "bob@example.com", "country": "Canada" },
       "assignment": true
     },
+    {
+      "subjectKey": "barbara",
+      "subjectAttributes": { "email": "barbara@example.com", "country": "canada" },
```

Comment:
regex match?

Reply:
Testing case-sensitivity: this should not match `Canada` (like bob above)